sim-lib/src/lnd.rs

```diff
@@ -24,6 +24,26 @@ pub struct LndNode {
     info: NodeInfo,
 }
 
+// FIXME: Do better with this error
+// TODO: We could even generalize this to parse any type of Features
+/// Parses the node features from the format returned by LND gRPC to LDK NodeFeatures
+fn parse_node_features(features: HashSet<&u32>) -> Result<NodeFeatures, String> {
```

Comment:
Just return `NodeFeatures` since this doesn't error?

Reply:
Yep, you're right. I mixed this with the previous commit